test/pummel/test-watch-file.js

```diff
@@ -34,8 +34,11 @@ fs.closeSync(fs.openSync(f, 'w'));
 let changes = 0;
 function watchFile() {
   fs.watchFile(f, (curr, prev) => {
+    // Make sure there is at least one watch event that shows a changed mtime.
+    if (curr.mtime <= prev.mtime) {
```

Comment:
Just for my understanding, this would never really be a strictly-less-than relation, and only ever larger or equal, but if we check for equality we might as well also check for less-than?

Reply:
For the most part, yes. It's a coherence check. Super unlikely edge case: It is possible for the time on a host to be adjusted backwards (to correct for it having been set incorrectly in the first place, for example) but that would be highly unusual of course. But...might as well accommodate it, I guess....